includes/EE_MCI_Controller.class.php

```diff
@@ -228,10 +228,13 @@ public function mci_submit_to_mailchimp($spc_obj, $valid_data)
                                 $reg_approved = true;
                             }
                         }
-
+                        // Pull the EE_Attendee object for the registration
                         $attendee = $registration->attendee();
-                        $att_email = $attendee->email();
-                        if (($attendee instanceof EE_Attendee) && ! in_array(
+                        // If no EE_Attendee object, skip the subcribe call to MailChimp.
+                        if(! $attendee instanceof EE_Attendee) {
+                            continue;
+                        }
```

Comment:
Missing the email var declaration after this
```suggestion
                        }
                        $att_email = $attendee->email();
```

Reply:
Silly copy pasta mistake on my part, thanks @knazart